packages/state-transition/src/signatureSets/index.ts

```diff
@@ -25,7 +25,7 @@ export * from "./blsToExecutionChange.js";
  */
 export function getBlockSignatureSets(
   state: CachedBeaconStateAllForks,
-  signedBlock: allForks.SignedBeaconBlock,
+  signedBlock: SignedBeaconBlock<ForkName.altair>,
```

Comment:
Why is this narrowed down to altair? The function iterates over multiple forks

```suggestion
  signedBlock: SignedBeaconBlock,
```

Reply:
The required block attributes are not available in `phase0`, so why can't use simple `SignedBeaconBlock` which includes that fork as well.